.circleci/config.yml

```diff
@@ -326,17 +333,24 @@ workflows:
       - test:
           codecov-upload: true
           test-suite-name: full
+          skip-conformance: "1"
       - test-window-post:
           go-test-flags: "-run=TestWindowedPost"
+          skip-conformance: "1"
           winpost-test: "1"
           test-suite-name: window-post
       - test-short:
           go-test-flags: "--timeout 10m --short"
           test-suite-name: short
+          skip-conformance: "1"
```

Comment:
maybe make the parameter opt-in, like 'winpost-test' above, so that it only needs to be set on the test itself, rather than skipped everywhere else.